x-pack/plugins/security_solution/public/common/components/landing_page/get_started/configs.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema, type TypeOf } from '@kbn/config-schema';
+
```

Comment:
I don't like having so much serverless stuff in the generic plugin. Can we add a big warning comment saying this file is temporary and should not be used?

Reply:
Yup, added comments on all the types here.